api/schema/azure_subscription_resource.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var (
+	azureSubscriptionResource = Resource{
+		TypeName: "azure_subscription",
+		Schema: resource_schema.Schema{
+			Version:     1,
+			Description: "Manages an Azure subscription in CloudSecure.",
+			Attributes: map[string]resource_schema.Attribute{
```

Comment:
Reorder the attributes to be in alphabetic order (except `id` which should remain on top).